### ISSUE TITLE
dbptk ui - update on both gives error

### ISSUE THREAD

**Issue by Laurira** [keeps/dbptk-ui#293]
When user changes metadata then there are now two options for updating.
![image](https://user-images.githubusercontent.com/3479156/123910624-65651a80-d983-11eb-9662-d844a4bd6c5c.png)

If user clicks "UPDATE ON BOTH" then error is displayed:
![image](https://user-images.githubusercontent.com/3479156/123910663-74e46380-d983-11eb-955d-48162a712919.png)

In the log I can not find more detailed description of that error:

solr_1    | 2021-06-30 06:09:10.251 INFO  (qtp1237762639-20764) [c:dbv-activity-logs s:shard1 r:core_node2 x:dbv-activity-logs_shard1_replica_n1] o.a.s.u.p.LogUpdateProcessorFactory [dbv-activity-logs_shard1_replica_n1]  webapp=/solr path=/update params={_stateVer_=dbv-activity-logs:4&waitSearcher=true&commit=true&softCommit=true&wt=javabin&version=2}{commit=} 0 13
dbptke_1  | 2021-06-30 06:09:10.258 ERROR 1 --- [nio-8080-exec-3] c.d.c.c.common.utils.AsyncCallbackUtils  : [192.168.65.64] AsyncCallback error - RESTException: Remote exception caused by GenericException: Could not update the database metadata information caused by NullPointerException: 
cas_1     | 2021-06-30 06:09:24,824 INFO [org.apereo.cas.services.AbstractServicesManager] - <Loaded [1] service(s) from [JsonServiceRegistry].>

**Comment by Laurira** [comment]
Still the same error:
![image](https://user-images.githubusercontent.com/3479156/127307825-d8789b23-b5c1-497c-97b1-e7a1d889b587.png)

[dbvtk-28-07-2021.log](https://github.com/keeps/dbptk-ui/files/6892313/dbvtk-28-07-2021.log)

**Comment by hmiguim** [comment]
@Laurira I found the culprit. I'll create a staging example for testing and provide it to you

**Comment by Laurira** [comment]
There is still some error
![image](https://user-images.githubusercontent.com/3479156/127461768-fb0925be-4180-46b0-b167-de42c9288685.png)

**Comment by Laurira** [comment]
Same error happened in the Enterprise as well.

**Comment by Laurira** [comment]
But it is working with some of our other databases. The error currently has occurred only with Sakila.

**Comment by Laurira** [comment]
I made docker compose down and up again so it should be clean sheet.
Now I tried to use database "Northwind".
Everything worked but after "Update" about 5 seconds there were displayed "RESTException" again. But the file was saved and new modification were stored.

When I see the log then
`
2021-08-13 06:12:31,717 [http-nio-8080-exec-3] INFO  c.d.m.siard.update.SIARDEditModule - Importing SIARD version 2.1
2021-08-13 06:12:31,769 [http-nio-8080-exec-3] INFO  c.d.model.reporters.Reporter - Changing description from 'unspecified' to 'uuendamisjama'
2021-08-13 06:12:31,769 [http-nio-8080-exec-3] INFO  c.d.model.reporters.Reporter - Metadata for description successfully updated
2021-08-13 06:12:31,900 [http-nio-8080-exec-3] INFO  c.d.m.siard.update.SIARDEditModule - Importing SIARD version 2.1
2021-08-13 06:12:31,948 [http-nio-8080-exec-3] INFO  c.d.model.reporters.Reporter - A report was generated with a listing of information that was modified during the conversion.
2021-08-13 06:12:31,948 [http-nio-8080-exec-3] INFO  c.d.model.reporters.Reporter - The report file is located at /dbvtk/reports/report-metadata-edition-6803f8ad-7b21-4ab7-a9bc-f4c6e14a945c-2021-08-13T06_12_23.792Z.md
`
And next line was probably related to the error
`
2021-08-13 06:12:31,978 [http-nio-8080-exec-7] ERROR c.d.c.c.c.utils.AsyncCallbackUtils - [192.168.65.64] AsyncCallback error - RESTException: Remote exception caused by GenericException: Could not update the database metadata information caused by NullPointerException:
`

**Comment by Laurira** [comment]
I guess the version 3.2 is even better?
![image](https://user-images.githubusercontent.com/3479156/131979018-1906b514-faa9-45b8-8ef5-f5781eeb04d3.png)

**Comment by hmiguim** [comment]
That is the docker-compose version. I meant you update the image version that is keeps/dbvtk:latest to keeps/dbvtk:v2.5.9

**Comment by Laurira** [comment]
Is version 2.5.9 with CAS?

**Comment by hmiguim** [comment]
CAS is a service provided independently of DBPTK Enterprise.

**Comment by Laurira** [comment]
Yes, I know that. You did create docker image of DBPTK Enterprise with CAS to us. So can I update only the DBPTK Enterprise in that image?

**Comment by hmiguim** [comment]
Yes, you can upgrade.

**Comment by Laurira** [comment]
I forgot to use word "how" instead of "if". How can I upgrade only DBPTK Enterprise in that image?

**Comment by hmiguim** [comment]
Change the property image in the docker-compos.yml file.

```docker-compose
(...)
services:
  dbptke:
    image: keeps/dbvtk:v2.5.9

(...)
```

**Comment by Laurira** [comment]
But I have image: keeps/dbvtk:latest there, so I have always the newest version?

**Comment by hmiguim** [comment]
It doesn't work like that, because if the docker has the image dbvtk:latest already in their docker images it won't pull the newer one with the same tag. 

Or you have to manually force it to happen with docker-compose pull or you have to remove the image and restart all.

**Comment by Laurira** [comment]
I used the version v2.5.9. I get no error but I can not click "Update on both" either.
![image](https://user-images.githubusercontent.com/3479156/132463449-35b50aad-8c8a-461d-a248-ea9f15fef0bb.png)
